Dockerfile.QA

```diff
@@ -61,6 +61,8 @@ RUN mkdir -p qa/common && \
     cp -r docs/examples/model_repository/simple qa/L0_simple_callback_example/models/. && \
     mkdir qa/L0_simple_string_example/models && \
     cp -r docs/examples/model_repository/simple_string qa/L0_simple_string_example/models/. && \
+    mkdir qa/L0_shared_memory/models && \
```

Comment:
You should call this L0_simple_shared_memory_example to match the naming of the other simple example tests.